scaaml/capture/aes/control.py

```diff
@@ -2,36 +2,25 @@
 
 from time import sleep
 
+from scaaml.capture.control import AbstractSControl
 
-class SControl:
+
+class SControl(AbstractSControl):
```

Comment:
Like the scope class, this one is also specific to Chipwhisperer hardware.
In general case, when we don't use the Chipwhisperer capture board, we can't reset or power cycle the chip.
So I suggest to use this refactoring to rename this class to make it more obvious that it's related to Chipwhisperer only